noir-projects/noir-contracts/contracts/token_blacklist_contract/src/types/balances_map.nr

```diff
@@ -6,14 +6,14 @@ use dep::aztec::{
 };
```

Comment:
Changed `T` to `Note` everywhere in this file as it's how we generally refer to a generic note type everywhere else in the codebase.